components/Features.js

```diff
@@ -10,89 +10,84 @@ const Wrapper = styled.div`
   display: flex;
   max-width: 82em;
   flex-direction: column;
-  text-align: center;
-  flex-wrap: wrap;
-  margin: 0 auto;
   h1 {
     font-size: 1.25rem;
     text-transform: uppercase;
+    text-align: center;
+    margin-top: 0;
   }
-  div {
-    display: inline-flex;
-    margin-top: 1.25rem;
-  `;
+`;
+
+const FeaturesList = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(17rem, 1fr));
+  grid-gap: 3rem;
+  margin-top: 2rem;
+`;
 
 const Feature = styled.div`
   display: flex;
   flex-direction: column;
-  width: 32em;
-  margin: 0 1.875rem;
+  align-items: center;
   line-height: 1.75;
-    .icon {
-      margin: 0.625rem auto;
-      margin-bottom: 1.25rem;
-      justify-content: center;
-    }
-    h1 {
-      margin: 0;
-      text-align: center;
-      font-size: 	1rem;
-      font-weight: 600;
-    }
-    .heart {
-      margin-top: 1.75rem;
-      }
-    }
-  `;
+  text-align: justify;
+  .icon {
+    margin-bottom: 1.25rem;
+  }
+  h2 {
+    margin: 0;
+    text-transform: uppercase;
+    font-size: 1rem;
+  }
+`;
+
 export default () => (
   <Wrapper>
     <h1>Amazing Features</h1>
-    <div>
+    <FeaturesList>
       <Feature>
-        <div className="icon">
-          <FontAwesomeIcon
-            icon={faShippingFast}
-            size="4x"
-            color="rgba(47,79,79, 0.6)"
-          />
-        </div>
-        <h1>Free shipping</h1>
+        <span className="fas fa-shipping-fast" />
```

Comment:
Does not this span need a closing tag?

Reply:
@nadezhda if there is no content inside the span (meaning `<span>content</span>`, then it should be written as `<span />`.